sensor_cfg:
  # 雷达参数
  lidar:
    translation:
      x: 0.0
      y: 0.0
      z: 0.416
    rotation:
      x: 1.0
      y: 0.0
      z: 0.0
      w: 0.0